packetbeat/protos/amqp/amqp_structs.go

```diff
@@ -162,66 +166,60 @@ const (
 )
 
 type amqpPrivateData struct {
-	Data [2]*AmqpStream
+	data [2]*amqpStream
 }
 
-type AmqpFrame struct {
-	Type    byte
-	channel uint16
+type amqpFrame struct {
+	Type frameType
+	// channel uint16  (frame channel is currently ignored)
```

Comment:
I commented it out because: a) it indirectly reflects the wire format. b) maybe in future this will be exposed?